README.rst

```diff
@@ -28,6 +28,7 @@ uses the ``xprocess`` fixture internally::
     # content of conftest.py
 
     import pytest
+    from pytest_xprocess import ProcessStarter
```

Comment:
This line is not working for me.
This should be `from xprocess import ProcessStarter`